packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -570,7 +570,13 @@ export class CallContext {
   public setDeviceManagerCameras(cameras: VideoDeviceInfo[]): void {
     this.setState(
       produce(this._state, (draft: CallClientState) => {
-        if ((!draft.deviceManager.cameras || draft.deviceManager.cameras.length === 0) && cameras.length > 0) {
+        if (
+          /** SDK initializes cameras with one dummy camera with value { id: 'camera:id', name: '', deviceType: 'USBCamera' } before camera permissions are granted.
```

Comment:
can we keep this comment above the if block so its all together?

Reply:
Sure.